Selenium.Axe/Selenium.Axe/HtmlReport.cs

```diff
@@ -420,7 +413,7 @@ private static void GetReadableAxeResults(AxeResultItem[] results, ResultType ty
 
                     foreach (var target in item.Target)
                     {
-                        content.AppendLine($"{HttpUtility.HtmlEncode(target)}");
+                        content.AppendLine($"{HttpUtility.HtmlEncode(target.ToString().Replace(",", " "))}");
```

Comment:
Is this change intended to be included? It doesn't feel related to the PR title.

It feels a bit dangerous to me; it's possible for an individual target to be, for example, `div[aria-label="foo, bar, and baz"]`

Reply:
Oddly it is related.
The way the formatting works means if we don't do this it will result in an invalid CSS Selector.
This change makes the HTML report match the Axe extension results.

Reply:
Selector doesn't really work as is so reverting